crates/ty_ide/src/hover.rs

```diff
@@ -2238,12 +2236,11 @@ def function():
             "#,
         );
 
-        // TODO: This should be `P@Alias (<variance>)`
         assert_snapshot!(test.hover(), @r"
-        typing.ParamSpec
+        P@Alias (bivariant)
```

Comment:
shouldn't this be contravariant? Is that worth a TODO comment?

E.g. when we hover over `T` here, we display it as being contravariant:

```py
from typing import Callable
type X[T] = Callable[[T], int]
```

Reply:
I'll add a TODO comment, I'm not actually sure about this but it does make sense for this to be Contravariant, the current variance is being inferred automatically.

Reply:
Yeah, I think it should either be contravariant or invariant... bivariant just seems definitely wrong, anyway 😆